crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -35,12 +33,6 @@ pub struct GeneralStaticCommittee<T: NodeType> {
     indexed_stake_table:
         BTreeMap<T::SignatureKey, <T::SignatureKey as SignatureKey>::StakeTableEntry>,
 
-    // /// The members of the committee
-    // committee_members: BTreeSet<T::SignatureKey>,
-    #[cfg(feature = "fixed-leader-election")]
-    /// The number of fixed leaders for gpuvid
-    fixed_leader_for_gpuvid: usize,
-
```

Comment:
Where did the fixed leader logic go to? I thought you mentioned we'd keep it, but I am missing where

Reply:
to recreate e.g. the single fixed leader, we'd just set `eligible_leaders` to that single node in `Memberships::new()`